README.md

```diff
@@ -34,6 +42,37 @@ opam install --deps-only $PKG
 After that, you can enter the folder containing the cloned sources and
 run the appropriate build command.
 
+## Using Opam inside Docker for Dev Work
+
+If you prefer to use [Docker] for development environments, you can
+do this as well:
+
+```bash
+make
+```
+
+This builds a [Docker] image with all Citrix Hypervisor toolstack
+libraries and components installed. Once built, it can be used to
+compile individual packages like [xenopsd]:
+
+
+```bash
+$ cd src/xenopsd
+$ docker run --rm -itv $PWD:/mnt xenserver/xs-opam:6.37.0
```

Comment:
The following would be more future proof

```
$ cd src/xs-opam
$ export XSOPAM_VERSION=$(git describe --abbrev=0 --tags)
$ cd ../xenopsd
$ docker run --rm -itv $PWD:/mnt "xenserver/xs-opam:$XSOPAM_VERSION"
```

Reply:
Yeah. I'm two minds about it. Less magic can help to see the picture better.

Reply:
Perhaps a good compromise is to leave it as 6.37.0, but mention that you can look at the following to find the correct tag

```
$ docker image ls xenserver/xs-opam
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
xenserver/xs-opam   6.37.0              1744d037eaec        2 days ago          4.51GB
xenserver/xs-opam   6.35.0              131c4fc20e0c        8 days ago          6.46GB
xenserver/xs-opam   6.31.0              b49d1b5311f4        3 months ago        6.36GB
...
```